### ISSUE TITLE
logy doesn't work for Curve and Path

### ISSUE THREAD

**Issue by basnijholt** [ioam/holoviews#553]
```python
%%opts Curve [logx=True logy=True]
holoviews.Curve((np.logspace(1, 3, 10), np.logspace(1, 3, 10)))
```

only results in an x-axis with log scale.

**Comment by basnijholt** [comment]
Right now you can only have one axis in logscale.
For me #554 fixes this.

**Comment by philippjfr** [comment]
Just merged #554, thanks for the fix!
